metabase/metabase
# Use list picker when searching a single value

Closes https://github.com/metabase/metabase/issues/49279

Switch back to the list filter instead of using the multi autocomplete input in dashboard filters.

### To verify


1. Go to Admin -> Table Metadata -> Products -> Title column and set it as Cateog
2. Create a dashboard that has a question relating to Products on it
3. Add a filter that is linked to the Title column
4. Set the filter type to `Search box` and the `A single value`
5. Try using the filter, and make sure it's using the list selector not the MultiAutoComplete 
input
5. This should also work when the filter type is set `Multiple values`

<img width="416" alt="Screenshot 2024-10-30 at 17 13 28" src="https://github.com/user-attachments/assets/6d6fda37-86a5-4a7d-be8e-d14fd7c40b60">
<img width="559" alt="Screenshot 2024-10-30 at 17 13 33" src="https://github.com/user-attachments/assets/3f42a7d5-2053-46ba-abae-0e6d339aaf98">
